src/screens/Introduction/Introduction.js

```diff
@@ -1,58 +1,38 @@
-import React from "react";
-import PropTypes from "prop-types";
-import { View, Text, Image } from "react-native";
-import { Button } from "components";
-import Indicator from "./Indicator";
-import * as utils from "./utils";
-import { styles } from "./styles";
+import React, { useState, useRef } from "react";
+import { View, Image, TouchableHighlight } from "react-native";
+import Button from "../../components/Button";
+import Slides from "./Slide/Slides";
+import { translate } from "../../i18n";
+import { I18_NAMESPACE, MAX_COUNT_SLIDER_ITEMS } from "./constants";
 
-const Introduction = props => {
-  const { navigation } = props;
-  const {
-    state: {
-      params: { introductionId, goBack }
-    }
-  } = navigation;
-  const onBackPress = () => {
-    if (goBack) goBack();
-    else navigation.goBack();
+import styles from "./styles";
 
-  };
-  const onNextPress = () => {
-    navigation.navigate(utils.getNavigationNextStep(introductionId));
+const imageClose = require("../../assets/cross.png");
+
+const Introduction = ({ navigation }) => {
+  const [sliderIndex, setSliderIndex] = useState(0);
+  const sliderRef = useRef(null);
+  const captionNextStep = translate(`${I18_NAMESPACE}next`);
+  const onSliderChanged = index => setSliderIndex(index);
+  const onPressClose = () => navigation.navigate("mainPage");
+  const onPressNext = () => {
+    if (sliderIndex >= MAX_COUNT_SLIDER_ITEMS - 1) {
+      navigation.navigate("mainPage");
```

Comment:
right after introduction I appear inside the application, as if I'm logged in user. is it normal flow?

Reply:
))) fixed